nservicebus/service-fabric/transaction-sharing.md

```diff
@@ -18,6 +18,12 @@ include: saga-business-data-access
 
 snippet: ServiceFabricPersistenceSynchronizedSession-Saga
 
-Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. When data needs to be persisted to a custom collection during handler or saga execution, a new transaction should be created using the `StateManager` property of the incoming session.
+Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. 
+
+### Using a custom transaction
+
+When data needs to be persisted to a custom collection during handler or saga execution, a new transaction should be created using the `StateManager` property of the incoming session.
```

Comment:
I think that an example scenario would help a lot. Like storing some information about the current execution no matter what's the outcome of it (if it succeeds or fails). I agree with @danielmarbach that currently this paragraph does not provide this.